website/pages/docs/calendar/plugins/calendar-controls.mdx

```diff
@@ -18,6 +18,50 @@ Change the calendar view.
 
 Set the date of the calendar. Format: `YYYY-MM-DD`.
 
+### `setFirstDayOfWeek(firstDayOfWeek: WeekDay): void`
+
+Set the first day of the week for the calendar. Value must be between 0 and 6 where 0 is Sunday, 1 is Monday etc.
+
+### `setLocale(locale: string): void`
+
+Set the locale of the calendar.
+
+### `setDefaultView(defaultView: ViewName): void`
+
+Set the default view for the calendar.
+
+### `setViews(views: [View, ...View]): void`
+
+Set the available views for the calendar. The views to be set must include the currently active view name.
+
+### `setDayBoundaries(dayBoundaries: DayBoundariesExternal): void`
+
+Set the day boundaries of the calendar.
+
+### `setWeekOptions(weekOptions: WeekOptions): void`
+
+Set the week options of the calendar.
+
+### `setCalendars(calendars: Record<string, CalendarType>): void`
+
+Set the available calendars to be displayed in the calendar.
+
+### `setIsDark(isDark: boolean): void`
```

Comment:
This one already has an API too on the CalendarApp class

Reply:
Good catch, will change :)

Reply:
Thanks! I'd rather stick to only 1 API for doing it, unless a lot of people start asking for `toggleTheme`